CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 ----------
 
+[9.3.0] - 2023-12-08
```

Comment:
```suggestion
[9.3.0] - 2024-01-04
```

Reply:
@robrap Apologies for the delay. Updated the date.